File: data/code_pairs/fortran/151.f90
program test_great_rsd
  implicit none

  ! Test cases for the great_rsd function
  call great_rsd(1, 2)
  call great_rsd(-1, -2)
  call great_rsd(100, 200)

contains

  ! Implementation of great_rsd that prints the sum of two numbers
  subroutine great_rsd(x, y)
    implicit none
    integer, intent(in) :: x, y
    print *, 'Sum of ', x, ' and ', y, ' is ', x + y
  end subroutine great_rsd

end program test_great_rsd

File: data/code_pairs/cpp/151.cpp
#include <iostream>

// Declaration of great_rsd in C++
void great_rsd(int a, int b) {
    std::cout << "Sum of " << a << " and " << b << " is " << (a + b) << std::endl;
}

// Main function to run test cases
int main() {
    // Test cases for the great_rsd function
    great_rsd(1, 2);
    great_rsd(-1, -2);
    great_rsd(100, 200);

    return 0;
}